doc/snmp/extension-to-physical-entity-mib.md

```diff
@@ -0,0 +1,149 @@
+# SONiC Entity MIB Extension #
+
+### Revision ###
+
+ | Rev |     Date    |       Author       | Change Description                |
+ |:---:|:-----------:|:------------------:|-----------------------------------|
+ | 0.1 |             |      Kebo Liu      | Initial version                   |
+
+
+
+## 1. Overview 
+
+The Entity MIB contains several groups of MIB objects, currently SONiC only implemented part of the entityPhysical group following RFC2737. The group contains a single table called "entPhysicalTable" to indentify the pysical components of the system. The MIB objects of "entityPhysical" group listed as below:
+
+	EntPhysicalEntry ::= SEQUENCE {
+		entPhysicalIndex          PhysicalIndex,
```

Comment:
can you describe the rule to calculate the index?

Reply:
added in section 4.4